src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,357 @@
+- [ConsenSys (PegaSys)](https://consensys.net/knowledge-base/ethereum-2/teku/) (the team behind [Teku](https://docs.teku.consensys.net/en/latest/) which uses [jvm-libp2p](https://github.com/libp2p/jvm-libp2p))
+- [Status.im](https://status.im/) (the team behind [Nimbus](https://nimbus.team/) which uses [nim-libp2p](https://github.com/status-im/nim-libp2p))
+- [ChainSafe Systems](https://chainsafe.io/) (the team behind [Lodestar](https://lodestar.chainsafe.io/) which uses [js-libp2p](https://github.com/libp2p/js-libp2p))
+- [Ethereum Foundation](https://ethereum.foundation/)
+
+During this long developmental cycle, libp2p was running in the above nodes on the Beacon Chain.
+And once the Beacon Chain was merged with the original execution, libp2p was fully integrated into the Ethereum mainnet!
+
+## How Ethereum Beacon Nodes use libp2p 🔍
+
+So which exact libp2p modules are utilized by Beacon Chain nodes?
+Per the specification, nodes use the following libp2p features:
+- [Transport](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#transport): TCP
+- [Encryption](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#encryption-and-identification): [Noise](https://github.com/libp2p/specs/tree/master/noise)
+- [Protocol Negotiation](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#protocol-negotiation): [Multistream Select 1.0](https://github.com/multiformats/multistream-select/#protocol)
+- [Stream Multiplexing](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#multiplexing): [mplex](https://github.com/libp2p/specs/tree/master/mplex) or [yamux](https://github.com/hashicorp/yamux/blob/master/spec.md)
+- [Messaging](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#the-gossip-domain-gossipsub): [GossipSub v1.1](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md)
+
+All of the modules above, aside from yamux, are mandatory.
+
+One notable omission is the [libp2p Kademlia DHT](https://github.com/libp2p/specs/tree/master/kad-dht); The networking specification instead specifies the use of discv5.
+The rationale is [provided here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#why-are-we-using-discv5-and-not-libp2p-kademlia-dht).
+
+The specific design details are out of scope for this post but are [described in great detail here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#design-decision-rationale).
+Please check it out to see why TCP was chosen (and not QUIC), why Noise, why GossipSub, etc.
+
+### Network Statistics 📊
+
+The number of networked Beacon Chain nodes (i.e. nodes running libp2p) is [around 5,000 at the current count](https://migalabs.es/beaconnodes) with this [client diversity](https://clientdiversity.org/#distribution).
```

Comment:
```suggestion
The number of networked Beacon Chain nodes (i.e., nodes running libp2p) is [around 5,000 at the current count](https://migalabs.es/beaconnodes) with this [client diversity](https://clientdiversity.org/#distribution).
```

Reply:
https://clientdiversity.org/#distribution doesn't show historical data.  As a result, maybe take a screenshot of  <img width="470" alt="image" src="https://user-images.githubusercontent.com/85411/210881807-c4bb61d3-b84f-4889-9ced-b29f25071607.png"> for posterity?